File: CRUDAccess/InvoiceRestAccess/Views/Model/CustomerView.cs
// ALLOWOVERWRITE-72DB9DEFF6267724AF8B697B1462B4EA

using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;


namespace Database.Views.Model
{

	[DataContract]
    public class CustomerView
    {
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "active")]            
                       public bool Active { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "addressPostZipCode")]                         
                       public string AddressPostZipCode { get; set; }
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "customerExteranlRef")]            
                       public string CustomerExteranlRef { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "customerId")]            
                       public int CustomerId { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "emailAddress")]            
                       public string EmailAddress { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "isCompany")]            
                       public bool IsCompany { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "name")]            
                       public string Name { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "phoneNumber")]            
                       public string PhoneNumber { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "taxNo")]            
                       public string TaxNo { get; set; }	    	
    						  
					   /// <summary>
                      /// Reference of PrimaryKey for Entity
                      /// </summary>
                      [DataMember(Name = "entityEntityId")]
					  public  int EntityEntityId { get; set; }   
    	
    	
    	
    	
   
    }
}

File: CRUDAccess/InvoiceWebClient/src/app/modules/admin/Customer/Models/CustomerView.ts
// ALLOWOVERWRITE-4B4141E844815DF26FE4A9A113495D83-0C45F0ADC5CF5D02D47C8C5C878074FA


    export interface ICustomerView {
      active: boolean;    	
      addressPostZipCode: string;		    			
      customerExteranlRef: string;    	
      customerId: number;    	
      emailAddress: string;    	
      isCompany: boolean;    	
      name: string;    	
      phoneNumber: string;    	
      taxNo: string;    	

					    entityEntityId : number;   
    }
    
    export class CustomerView implements ICustomerView {
      active: boolean;    	
      addressPostZipCode: string;		    			
      customerExteranlRef: string;    	
      customerId: number;    	
      emailAddress: string;    	
      isCompany: boolean;    	
      name: string;    	
      phoneNumber: string;    	
      taxNo: string;    	

					    entityEntityId : number ;   


	  constructor() {
	  }
    }

